tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -204,7 +204,7 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                           RKE_VERSION = "v1.5.0-rc4"
                         }
                         if (!env.RANCHER_K3S_VERSION) {
-                          RANCHER_K3S_VERSION = "v1.27.6+k3s1"
+                          RANCHER_K3S_VERSION = "v1.28.2+k3s1"
```

Comment:
We're using `v1.28.6+k3s2` everywhere, so this would be updated to `v1.28.6+k3s2` too